LoopKit/DeviceManager/CGMManager.swift

```diff
@@ -19,6 +19,14 @@ public enum CGMResult {
     case error(Error)
 }
 
+public struct CGMManagerStatus {
```

Comment:
Why not add another `enum` to `CGMResult`, `hasValidSensorSession` with a `Bool` payload?  Wouldn't that obviate the need for an additional struct and `delegate` call?

Reply:
I think it makes sense to keep updates that are not associated with individual sensor readings separate. The CGM result is used for reporting readings that are automatically read, and also when Loop wants to try to fetch readings. I think we should keep that field scoped to those actions.

Reply:
I think this boils down to the above issue with `Result`, `Status`, and `State` being so similar.  Renaming them (or, at least, documenting [comments] exactly what their difference is) would help, I think.  Thanks.